client/reader/stream/empty-search-rec.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { RelatedPostCard } from 'blocks/reader-related-card-v2';
+import { recordTrackForPost, recordAction } from 'reader/stats';
+import { EMPTY_SEARCH_RECOMMENDATIONS, } from 'reader/follow-button/follow-sources';
+
+export default function EmptySearchRecommendedPosts( { post } ) {
+	function handlePostClick() {
+		recordTrackForPost( 'calypso_reader_recommended_post_clicked', post, {
+			recommendation_source: 'empty-search',
+		} );
+		recordAction( 'search_page_rec_post_click' );
+	}
+
+	function handleSiteClick() {
+		recordTrackForPost( 'calypso_reader_recommended_site_clicked', post, {
+			recommendation_source: 'empty-search',
+		} );
+		recordAction( 'search_page_rec_site_click' );
+	}
+
+	const site = { title: post.site_name, };
```

Comment:
I want to avoid modifying functionality as much as possible in this PR, so lets address it in a future one